### ISSUE TITLE
Are there any plans for the python version of this algorithm?

### ISSUE THREAD

**Issue by JingYansen** [phflot/flow_registration#22]

**Comment by phflot** [comment]
@JingYansen yes, I am currently working on a complete python framework that is compatible with MATLAB version and also includes an iteration of the compensation approach with even higher accuracy. Please ping me by the end of February, I might be able to give you access or link you to the repo by then.
Best,
Philipp

**Comment by JingYansen** [comment]
Thank you so much! Looking forward to the python framework!

**Comment by JingYansen** [comment]
@phflot Hi! I‘d like to inquire about any progress with the Python version?

**Comment by abrazhe** [comment]
@phflot Hi! Any news about the python version, maybe possibility to join in with development?
